webhook/handlers/UserController.js

```diff
@@ -0,0 +1,177 @@
+const db = require('../../models');
+
+class UserRegistrationController {
+  
+  async newUserIntent(agent, body) {
```

Comment:
remove Intent from name. Use `createNewUser` as function name. Also make changes to index.js